### ISSUE TITLE
Warning / error during install

### ISSUE THREAD

**Issue by fylla12** [astrada/google-drive-ocamlfuse#249]
Cut past from running   ->   "opam install google-drive-ocamlfuse"

∗  installed conf-which.1
∗  installed ocamlbuild.0
∗  installed conf-zlib.1
[ERROR] The compilation of conf-gmp failed at "sh -exc cc -c $CFLAGS -I/usr/local/include test.c".
∗  installed camlidl.1.05
∗  installed ocamlfind.1.7.1

...
..

#=== ERROR while installing conf-gmp.1 ========================================#
# opam-version 1.2.2
# os           linux
# command      sh -exc cc -c $CFLAGS -I/usr/local/include test.c
# path         /home/datb/.opam/system/build/conf-gmp.1
# compiler     system (4.02.3)
# exit-code    1
# env-file     /home/datb/.opam/system/build/conf-gmp.1/conf-gmp-23408-0ee25e.env
# stdout-file  /home/datb/.opam/system/build/conf-gmp.1/conf-gmp-23408-0ee25e.out
# stderr-file  /home/datb/.opam/system/build/conf-gmp.1/conf-gmp-23408-0ee25e.err
### stderr ###
# + cc -c -I/usr/local/include test.c
# test.c:1:17: fatal error: gmp.h: No such file or directory
#  #include <gmp.h>
#                  ^
# compilation terminated.

**Comment by astrada** [comment]
You should install external dependencies before installing this app:

    opam install depext
    opam depext google-drive-ocamlfuse

**Comment by fylla12** [comment]
I think I have figured it out. For  a debian test system I tried to install
it from the source code.

I used the instruction found at

github.com/astrada/google-drive-ocamlfuse/wiki/
How-to-install-from-source-on-Debian-Jessie

The steps
"
                                  opam install depext
                                  opam depext google-drive-ocamlfuse
"

was not mentioned there.  I included those steps and the following did work.


opam init
opam update
opam install depext
opam depext google-drive-ocamlfuse
opam install google-drive-ocamlfuse
. /home/*user*/.opam/opam-init/init.sh > /dev/null 2> /dev/null || true

But the where binary ended up was not clear because
"~/.opam//bin/google-drive-ocamlfuse" hinted in the instruction did not
exist.

At my system it ended up as
/home/XXXX/.opam/system/bin/google-drive-ocamlfuse

I could mount my google drive.

Some of the hang issues I had experienced seems to be gone and not to be
missed.
